### ISSUE TITLE
Onboarding screen is displayed when opening the app that already has profile

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#21034]
Reproduced issue in release 2.30 on Android 12, Samsung Galaxy A52

I have faced this issue only once, there are no steps for reproduction.

Among the details that might be useful:
1. I had both PR and release builds installed on my device.
2. Release app has been in background for sometime. 
3. New release build has been published recently before I faced the issue. I have not updated to this new built yet when issue happened. 

[onboardingscreen.log.zip](https://github.com/user-attachments/files/16588039/onboardingscreen.log.zip)

https://github.com/user-attachments/assets/6e046eb0-d7bc-434e-a92b-f1e9467aa877

### Additional Information

- Status version: release 2.30
- Operating System: Android

**Comment by Parveshdhull** [comment]
Thank you @pavloburykh for providing logs, they were quite [helpful](https://github.com/status-im/status-mobile/issues/20930#issuecomment-2288250551).

Closing this issue as duplicate of https://github.com/status-im/status-mobile/issues/20930, as other one has more context.
